src/css/opsz-demo.css

```diff
@@ -177,3 +177,16 @@
 		right: 0;
 	}
 }
+
+@media screen and (max-width: 48em) {
+	.opsz-medium,
+	.opsz-medium-with,
+	.opsz-large,
+	.opsz-large-with {
+		font-size: 1.5rem;
```

Comment:
I wouldn't flatten them all to one font size, as you can't discern the title:

<img width="579" alt="image" src="https://user-images.githubusercontent.com/4570664/73934956-c9df8f80-48df-11ea-9dd5-75267e69e701.png">

Reply:
Ah true. Will make the title a little bit bigger.